frame/contracts/src/schedule.rs

```diff
@@ -644,6 +647,7 @@ impl<T: Config> Default for HostFnWeights<T> {
 				1
 			)),
 			debug_message: to_weight!(cost_batched!(seal_debug_message)),
+			debug_message_per_byte: to_weight!(cost_byte_batched!(seal_debug_message_per_kb)),
```

Comment:
We don't batch in the benchmark.

```suggestion
			debug_message_per_byte: to_weight!(cost_byte!(seal_debug_message_per_kb)),
```

Reply:
True. Fixed.